core/scripts/chaincli/handler/scrape_node_config.go

```diff
@@ -0,0 +1,270 @@
+package handler
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"sync"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/google/go-cmp/cmp"
+	"github.com/manyminds/api2go/jsonapi"
+
+	"github.com/smartcontractkit/chainlink/v2/core/cmd"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+)
+
+type CSAKeyInfo struct {
+	NodeName    string `json:"nodeName"`
+	NodeAddress string `json:"nodeAddress"`
+	PublicKey   string `json:"publicKey"`
+}
+
+func (ci *CSAKeyInfo) Equals(ci2 *CSAKeyInfo) bool {
+	return ci.PublicKey == ci2.PublicKey && ci.NodeAddress == ci.NodeAddress
+}
+
+type NodeInfo struct {
+	AdminAddress          common.Address `json:"adminAddress"`
+	CSAKeys               []*CSAKeyInfo  `json:"csaKeys"`
+	DisplayName           string         `json:"displayName"`
+	Ocr2ConfigPublicKey   []string       `json:"ocr2ConfigPublicKey"`
+	Ocr2ID                []string       `json:"ocr2ID"`
+	Ocr2OffchainPublicKey []string       `json:"ocr2OffchainPublicKey"`
+	Ocr2OnchainPublicKey  []string       `json:"ocr2OnchainPublicKey"`
+	NodeAddress           []string       `json:"nodeAddress"`
+	OcrSigningAddress     []string       `json:"ocrSigningAddress"`
+	PayeeAddress          common.Address `json:"payeeAddress"`
+	PeerId                []string       `json:"peerId"`
```

Comment:
Should the json annotation here be `peerID` to be consistent with `ocr2ID`?

Reply:
in fact, according to rddtool style, this should be `ocr2Id`. so i am working on a fix to clean up those before i fix this.

Reply:
done